skills_ml/ontologies/esco.py

```diff
@@ -0,0 +1,81 @@
+from .base import Competency, Occupation, CompetencyOntology
+import logging
+import csv
+import requests
+
+lang = '&language=en'
+api_tax = 'https://ec.europa.eu/esco/api/resource/taxonomy?uri=http://data.europa.eu/esco/concept-scheme/isco&language=en'
+
+
+class Esco(CompetencyOntology):
+    def __init__(self):
+        super().__init__()
+        self.is_built = False
+        self.competency_framework.name = 'esco_skills'
+        self.competency_framework.description = 'ESCO Knowledge, Skills/Competencies'
+        self._build()
+    
+    def _build(self):
+        if not self.is_built:
+            reqTax = requests.get(api_tax)
+            getConcepts = reqTax.json()['_links']['hasTopConcept']
+            for concept in getConcepts:
+                _conceptProcessing(concept['href'])
```

Comment:
This doesn't run. All of these instance method invocations need `self.`, like `self._conceptProcessing(concept['href'])` here